src/mmw/js/src/modeling/modificationConfigUtils.js

```diff
@@ -2,6 +2,15 @@
 
 var modificationConfig = require('./modificationConfig.json');
 
+// Useful for testing.
+function setConfig(config) {
```

Comment:
I thought `setConfig` was unambiguous because it's in the `modificationConfigUtils` module.